reana_client/utils.py

```diff
@@ -206,7 +204,19 @@ def _validate_reana_yaml(reana_yaml):
         raise e
     except ValidationError as e:
         logging.info("Invalid REANA specification: {error}".format(error=e.message))
-        raise e
+        if e.validator == "additionalProperties":
+            invalid_section = (
+                ".".join(e.path[1], e.message.split("'")[1])
+                if len(e.path) > 1
+                else e.message.split("'")[1]
+            )
```

Comment:
For simplicity, perhaps we can just use `e.message` here? Unless I'm missing something, the messages are quite descriptive:

> "Additional properties are not allowed ('typ' was unexpected)"